### DISCUSSION TITLE
浅浅浅析财政数据--税源稳定否 | /project/fiscal1/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#576] [Announcements]
## 浅浅浅析财政数据--税源稳定否

https://yuanfan.rbind.io/project/fiscal1/

**Comment by XiangyunHuang** [comment]
关于税源是否稳定，我想到一个分析思路，就某种税（可以看最主要的那个），分析它的结构，用统计中的分类数据分析方法（比如卡方检验或者比例检验），随时间看结构变化是否显著，显著的话，意味着不稳定，然后再看是向好还是向坏，需要分析影响结构最显著的税，以及这个税对应的什么行业/产业，再看影响这个税的主要因素（可能有政策因素，那正好可以看政策发生前后的对比）。

**Reply by XiangyunHuang** [reply]
国家统计局公布历年各项税收数据 https://www.stats.gov.cn/sj/ndsj/2024/indexch.htm
